### DISCUSSION TITLE
Bulk action of the 'overdue' parameter

### DISCUSSION THREAD

**Discussion by GuilhermePesoa** [amidaware/tacticalrmm#1728] [Q&A] [answered]
Hello!
I intend to adjust the time without communication parameter so that the endpoint receives the 'overdue' state.

I would like to adjust this on all endpoints
Is there an automated way to do this?
Could you help me find this path? Thanks!

**Comment by dinger1986** [comment, marked as answer]
sure https://docs.tacticalrmm.com/management_cmds/#bulk-update-agent-offline-overdue-time

**Reply by GuilhermePesoa** [reply]
Thank you for your help!
